.circleci/config.yml

```diff
@@ -60,6 +60,25 @@ jobs:
           fail_only: true
           failure_message: Nightly auth tests failed!
           webhook: $SLACK_WEBHOOK
+  uptime_monitor_waf:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/buildpack-deps:18.04-browsers
+        auth:
+          username: dockstoretestuser
```

Comment:
Thought, is this a good time to start experimenting with contexts?

Reply:
I think this using a context.